src/networkmessage.cpp

```diff
@@ -143,7 +143,7 @@ void NetworkMessage::addItem(const Item* item)
 	if (it.isContainer()) {
 		addByte(0x00); // assigned loot container icon
 		// quiver ammo count
-		const Container* container = item->getContainer();
+		auto container = item->getContainer();
```

Comment:
Could this be `const auto`
```suggestion
		const auto container = item->getContainer();
```